### PR TITLE
[4] Delete lots of code

### PR DESCRIPTION
**Pull request by PhilETaylor**
Code review.

Delete all the things. Make them smaller. Prettier. Modern. More Professional. Less Code = Less Bugs

- 'return' is unnecessary as the last statement in a method
- Ternary expression replaced with short version
- Unnecessary 'return' statement
- 'continue' is unnecessary as the last statement in a loop

### CODE REVIEW

File: libraries/src/Document/FeedDocument.php [hunk 1]
```diff
@@ -203,7 +203,7 @@ public function render($cache = false, $params = array())
 		$type = CmsFactory::getApplication()->input->get('type', 'rss');
 
 		// Instantiate feed renderer and set the mime encoding
-		$renderer = $this->loadRenderer(($type) ? $type : 'rss');
+		$renderer = $this->loadRenderer(($type) ?: 'rss');
```
Comment (Quy) [inline libraries/src/Document/FeedDocument.php]:
```suggestion
		$renderer = $this->loadRenderer($type ?: 'rss');
```

File: libraries/src/Form/Form.php [hunk 2]
```diff
@@ -302,7 +302,7 @@ public function getFieldset($set = null)
 		{
 			// Get the field groups for the element.
 			$attrs = $element->xpath('ancestor::fields[@name]/@name');
-			$groups = array_map('strval', $attrs ? $attrs : array());
+			$groups = array_map('strval', $attrs ?: array());
```
Comment (Quy) [inline libraries/src/Form/Form.php]:
```suggestion
			$groups = array_map('strval', $attrs ?: []);
```
Repeat

Comment (PhilETaylor) [reply libraries/src/Form/Form.php]: Thanks.

File: libraries/src/Feed/FeedParser.php [hunk 3]
```diff
@@ -63,7 +63,7 @@ abstract class FeedParser
 	public function __construct(\XMLReader $stream, InputFilter $inputFilter = null)
 	{
 		$this->stream      = $stream;
-		$this->inputFilter = $inputFilter ? $inputFilter : InputFilter::getInstance(array(), array(), 1, 1);
+		$this->inputFilter = $inputFilter ?: InputFilter::getInstance(array(), array(), 1, 1);
```
Comment (Quy) [inline libraries/src/Feed/FeedParser.php]:
```suggestion
		$this->inputFilter = $inputFilter ?: InputFilter::getInstance([], [], 1, 1);
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7d30e21760b02558d8da5ae4cf575ae3586ad2ce<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36415">issues.joomla.org/tracker/joomla-cms/36415</a>.</sub>

**Comment by PhilETaylor** [comment]
Thanks for testing @Quy - appreciate it.

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 7d30e21760b02558d8da5ae4cf575ae3586ad2ce<br /><br />Nice cleanup! thanks @PhilETaylor <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36415">issues.joomla.org/tracker/joomla-cms/36415</a>.</sub>

**Comment by RickR2H** [comment]
RTX<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36415">issues.joomla.org/tracker/joomla-cms/36415</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**